source/Octodiff/Core/BinaryDeltaReader.cs

```diff
@@ -12,10 +12,11 @@ public class BinaryDeltaReader : IDeltaReader
         private byte[] expectedHash;
         private IHashAlgorithm hashAlgorithm;
         private bool hasReadMetadata;
+        private readonly int defaultBufferSize = 4 * 1024 * 1024;
```

Comment:
Nit: use a `const` here

Reply:
Thanks Adam, this has been addressed [here](https://github.com/OctopusDeploy/Octodiff/pull/39/commits/5b886de05dd2fa0ca1118cdefbc894ed6652e693).